homing_missiles.py

```diff
@@ -109,12 +109,22 @@ def draw_entities(self, screen):
         pygame.draw.rect(screen, self.main_character_color_, main_character_rect)
         self.draw_friendly_projectiles(screen)
 
+    def determine_enemy_color(self, enemy):
+        enemy_type = enemy.get_type()
+        if enemy_type == "BasicGrunt":
+            return YELLOW
+        elif enemy_type == "NotSoBasicGrunt":
+            return WHITE
+
+        return RED
+
     def draw_target_tags(self, screen):
         for enemy in self.universe_.enemies():
+            enemy_color = self.determine_enemy_color(enemy)
```

Comment:
This is something we're doing twice, but I don't have any reasonable solutions to not do it twice. We should make a note of this but not something we need to change for the prototype.